src/main/java/com/zufar/icedlatte/review/endpoint/RatingEndpoint.java

```diff
@@ -0,0 +1,38 @@
+package com.zufar.icedlatte.review.endpoint;
+
+import com.zufar.icedlatte.openapi.dto.AddNewMarkToProductRequest;
+import com.zufar.icedlatte.openapi.dto.RatingDto;
+import com.zufar.icedlatte.review.api.RatingUpdater;
+import com.zufar.icedlatte.security.api.SecurityPrincipalProvider;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping(value = RatingEndpoint.RATING_URL)
+public class RatingEndpoint {
+
+    public static final String RATING_URL = "/api/v1/rating/product";
```

Comment:
I'd consider using `/api/v1/products/rating` and `/api/v1/products/review`, what do you think?

Reply:
Zufar insists on `api/v1/products/{productId}/rating` https://github.com/Sunagatov/Iced-Latte/pull/246#discussion_r1493787143